tools/cell_census_builder/experiment_builder.py

```diff
@@ -265,7 +265,7 @@ def populate_presence_matrix(self) -> None:
             # sanity check
             assert len(self.presence) == self.n_datasets
 
-            max_dataset_joinid = max(self.presence.keys())
+            max_dataset_joinid = max(d.soma_joinid for d in datasets)
```

Comment:
this is wrong - it is going to add datasets which do not exist in the organism's experiment.